zeppelin-web/src/app/notebook/paragraph/paragraph.controller.js

```diff
@@ -755,8 +755,13 @@ function ParagraphCtrl($scope, $rootScope, $route, $window, $routeParams, $locat
       }
 
       autoAdjustEditorHeight(_editor);
-      angular.element(window).resize(function() {
-        autoAdjustEditorHeight(_editor);
+
```

Comment:
@Savalek instead of this; could we have done `angular.element(window).unbind('resize');` in `$scope.$on('$destroy', function() {`, would it yield same result?

Reply:
@prabhjyotsingh, in this case, the replacement of this option is possible. Change it to your version?

Reply:
Sure, as I think this is bit more readable.

Reply:
Сhanged it.